public/main.css

```
@@ -0,0 +1,111 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    text-decoration: none;
+    list-style: none;
+}
+.app{
+    color:black;
+    display: grid;
+    grid-template-areas:
+    "app_nav app_content app_content"
+    "app_bottom_bar app_bottom_bar app_bottom_bar";
+    grid-template-columns: 240px 1fr;
+    grid-template-rows: 90% 10%;
+    height: 100vh;
+    width: 100%;
+    font-family: spotify-circular, Helvetica, Arial, sans-serif;
+}
+.app_nav{
+    display: flex;
+    flex-direction: column;
+    grid-area:app_nav ;
+    background-color: black;
+}
+.spotify_logo{
+    color: white;
+    width: 130px;
+    height: 40px;
+}
+.nav_element{
+    margin: 25px 25px;
```

Можно сократить запись до:
```suggestion
    margin: 25px;
```